var/spack/repos/builtin/packages/metall/package.py

```diff
@@ -39,7 +41,7 @@ class Metall(CMakePackage):
     version("0.9", sha256="2d7bd9ea2f1e04136050f210884445a9e3dcb96c992cf42ff9ea4b392f85f927")
 
     depends_on("cmake@3.10:", type="build")
-    depends_on("boost@1.64:", type=("build", "link"))
+    depends_on("boost@1.75:", type=("build", "link"))
```

Comment:
Does this change apply to all of the listed versions or for some version on?  If the latter, then you could keep the original `depends_on` directive and add a new one with a `when` argument (https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-dependencies), e.g., if it applies to `0.24` on then the dependencies could be:

```suggestion
    depends_on("boost@1.64:", type=("build", "link"))
    depends_on("boost@1.75:", when="@0.24:", type=("build", "link"))
```

Reply:
Yes, the change applies to all of the versions. We are thinking of stopping officially supporting any versions less than 1.75 for Boost.